docs/Ruleset.md

```diff
@@ -158,6 +162,8 @@ V114 | MAJOR | Trait private method parameter default removed
 V115 | MAJOR | Trait public method parameter default value changed
 V116 | MAJOR | Trait protected method parameter default value changed
 V117 | MAJOR | Trait private method parameter default value changed
+V152 | PATCH | Trait method renamed (case only)
```

Comment:
This should be split into 3 rules, one for each visibility level (as it is done for all other rules even if they are all the same value).